test/cypress/integration/22-collective.edit.test.js

```diff
@@ -142,9 +142,9 @@ describe('edit collective', () => {
   });
 
   it('enables VAT', () => {
-    cy.contains('[data-cy="country-select"]', 'Please select your country').click();
-    cy.contains('[data-cy="select-option"]', 'Belgium').click();
+    cy.get('input[id=geosuggest__input]').type('Belgium').wait(750).type('{downArrow}{enter}');
```

Comment:
Such a `wait` is dangerous; if the request takes more than 750ms the test will fail.

For improved reliability, wait for an element that will only be there after the query completes, ex:

```es6
cy.get(...).type('Belgium');
cy.get('#google-thing .search-result').click()
```

Reply:
Good call, fixed this! Also updated a similar test in `23-event.create.test.js`